src/requirements_dev.in

```diff
@@ -1,17 +1,9 @@
-# Base website
--r ./requirements.txt
+-r ./requirements.in
 
-# Tools for maintaining requirements.txt:
-pur == 7.0.0
-
-# Useful extra developer packages:
-black == 22.10.0
-isort == 5.10.1
-pytest-sugar == 0.9.5
-termcolor == 2.0.1
-pre-commit == 2.20.0
-py         == 1.11.0
-
-# Debugging
-django-debug-toolbar == 3.7.0
-django-extensions == 3.2.1
+black
+isort
+pytest-sugar==0.9.6
+termcolor
+pre-commit 
+django-debug-toolbar
+django-extensions
```

Comment:
I think there should be versions here. Otherwise, every `make requirements` may produce a different `requirements_dev.txt` and black/isort may also change their rules.

Reply:
done